sql/update-2024-08-11_2.sql

```diff
@@ -1,13 +1,23 @@
-CREATE TABLE `storage_team` (
+CREATE TABLE IF NOT EXISTS `storage_team` (
   `accountId` bigint(20) NOT NULL,
   `itemId` bigint(20) NOT NULL,
-  `position` int(11) NOT NULL
+  `position` int(11) NOT NULL,
+  PRIMARY KEY (`accountId`, `itemId`),
+  KEY `itemId` (`itemId`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;
 
+-- Drop the first foreign key if it exists
+-- ALTER TABLE `storage_team`
+-- DROP FOREIGN KEY `storage_team_ibfk_1`;
+
+-- Drop the second foreign key if it exists
+-- ALTER TABLE `storage_team`
+-- DROP FOREIGN KEY `storage_team_ibfk_2`;
+
+-- Add the first foreign key constraint
 ALTER TABLE `storage_team`
-  ADD PRIMARY KEY (`accountId`,`itemId`),
-  ADD KEY `itemId` (`itemId`);
+ADD CONSTRAINT `storage_team_ibfk_1` FOREIGN KEY (`accountId`) REFERENCES `accounts` (`accountId`) ON DELETE CASCADE ON UPDATE CASCADE;
```

Comment:
ive modified these two SQL because i was getting errors about it having two primary keys btw